frameworks/g-cloud-9/questions/declaration/GAAR.yml

```diff
@@ -1,3 +1,4 @@
+name: Why a tax return was incorrect
```

Comment:
"Why" seems weird for a Yes/No question.  Could it be something like "Incorrect tax return"?